src/main/java/codeu/model/data/User.java

```diff
@@ -56,8 +65,33 @@ public User(UUID id, String name, Instant creation, String hashedPassword) {
     this.creation = creation;
     this.hashedPassword = hashedPassword;
     this.about = "";
+    this.taggedConversations = new HashMap<>();
+    this.conversationTags = new HashMap<>();
   }
 
+  /**
+   * Adds a Tag to a Conversation.
```

Comment:
Are theses conversations that a user has created themselves?

Reply:
Nope, these are the Conversations in ConversationStore

Reply:
I'm a little confused, when will we call this method?